### ISSUE TITLE
Switching audio tracks or subtitles doesn't work for some files

### ISSUE THREAD

**Issue by oekaki** [mpc-qt/mpc-qt#23]
mpc-qt v22.02, Windows 11

Neither the right click menu or the hotkeys work.

Sample: https://0x0.st/o8kh.mkv

**Comment by cmdrkotori** [comment]
Turn this option off.
![Screenshot_20220220_082109](https://user-images.githubusercontent.com/65291121/154821019-b788da9b-0306-4cb2-ae3c-238d22336b21.png)
I've changed the default so this shouldn't happen any more.

**Comment by itsmattkc** [comment]
I just ran into this issue. Unfortunately there hasn't been a release tag since this default was changed, so most users (who aren't compiling from source) are still going to get the old default. It might be good to make another release soon?

**Comment by ChexGuy** [comment]
I've encountered strange behavior regarding audio tracks that aren't named, like with nvidia shadowplay/share recordings. I _used_ to be able to switch these during playback, but now the track just switches itself back to the first one automatically. It doesn't even seem to be a version thing, as v20.10 and v23.12 (and the two in between) yield the same outcome.

The one thing I can point to is that I began using audio routing software (VoiceMeeter) and setting mpc-qt to use it as the audio device.

Switching audio tracks that are named doesn't seem to have any issues.
